doc/releases/release-notes-4.2.rst

```diff
@@ -85,6 +85,8 @@ New APIs and options
 
     * :c:func:`bt_le_get_local_features`
 
+    * LE Connection Subrating is no longer experimental.
+
```

Comment:
would not bother adding it in the release notes, was tentatively focused on feature (or de-feature I suppose for the deprecated stuff)